packages/modular-scripts/package.json

```diff
@@ -37,14 +37,15 @@
     "@schemastore/tsconfig": "0.0.9",
     "@svgr/core": "6.2.1",
     "@svgr/webpack": "6.2.1",
+    "@swc/core": "^1.3.46",
+    "@swc/jest": "^0.2.26",
     "@types/case-sensitive-paths-webpack-plugin": "^2.1.6",
     "@types/lodash": "^4.14.191",
     "@types/micromatch": "4.0.2",
     "@types/node": "^18.11.18",
     "@types/npmcli__arborist": "^5.6.0",
     "@types/postcss-normalize": "^9.0.1",
     "@types/shell-quote": "^1.7.1",
-    "@types/webpack": "^5.28.0",
```

Comment:
not sure why this got removed? - don't remember doing it. 
nothing broke so I'm wondering if the types came with webpack already and this was unnecessary